src/views/credits/credits.jsx

```diff
@@ -157,28 +157,18 @@ var Credits = React.createClass({
                     </li>
 
                     <li>
-                        <img src="//cdn.scratch.mit.edu/get_image/user/36977_170x170.png" alt="Connor Avatar" />
-                        <span className="name">Connor Hudson</span>
-                    </li>
-
-                    <li>
-                        <img src="//cdn.scratch.mit.edu/get_image/user/14110644_170x170.png" alt="Lily Avatar" />
-                        <span className="name">Lily Kim</span>
-                    </li>
-
-                    <li>
-                        <img src="//cdn.scratch.mit.edu/get_image/user/13639421_170x170.png" alt="Tauntaun Avatar" />
-                        <span className="name">Tauntaun Kim</span>
-                    </li>
-
-                    <li>
                         <img src="//cdn.scratch.mit.edu/get_image/user/373646_170x170.png" alt="Dalton Avatar" />
                         <span className="name">Dalton Miner</span>
                     </li>
-
+                    
+                    <li>
+                        <img src="//cdn2.scratch.mit.edu/get_image/user/default_60x60.png" alt="Joel Avatar" />
+                        <span className="name">Joel Gritter</span>
```

Comment:
Members on the credits page should be in alphabetical order by last name.

Reply:
Ah, That makes sense 👍